controllers/chat/getChannelDetail.js

```diff
@@ -4,6 +4,7 @@ const {User} = require('../../databases/models');
 const {ResponseSuccess} = require('../../utils/Responses');
 const UsersFunction = require('../../databases/functions/users');
 const BetterSocialCore = require('../../services/bettersocial');
+const {MESSAGE_TYPE} = require('../../helpers/constants');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `MESSAGE_TYPE` is a good addition to the code. It helps in maintaining consistency and avoiding hard-coded values.
```diff
+ const {MESSAGE_TYPE} = require('../../helpers/constants');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->